packages/web/lib/page.js

```diff
@@ -26,8 +77,22 @@ const getPageScope = (parent) => ({
         scope: helpScope,
         key,
         functions: {
+          config:
+            "Takes in a ruleSet and overrides the set function so that any sets within the ruleSet set config variables",
           click:
             "Takes in a selector and clicks the argument indicated by the selector",
+          "@string":
+            "<key> <selector>: Sets a key on the data object in nearest page scope to a string value specified by selector",
```

Comment:
don't explain this using "scope" because that's an implementation detail of the interpreter/runtime, and does not mean anything to the user

Reply:
The issue is that we need some way to refer to scope, even if we don't call it scope. Even to understand how @config works require some knowledge of scope. This is also a good example of a function that requires a knowledge of scope. What happens if I use this in a for loop, what happens if I have nested page calls with @ functions in each with the same name. We can reword to obfuscate scope but we're still going to indirectly be talking about it I think.

What type of wording would you prefer?

Reply:
hmm, I think you're right. I ideally wanted it to be completely hidden from the user, but I see why that's hard when thinking about control flow and rulesets

let's use scope. You/Abel can help add to the Wiki about what a scope means and how it works

I think, regarding terminology, we want to be very clear about **Rule, RuleSet, Scope and Runtime**
(runtime is just the context in which a Rule or RuleSet is evaluated).

See if you can add to the wiki later, maybe a new page, that has definitions for each of these concepts. A human readable one, and then maybe, an optional, more detailed explanation that is useful for langauge/library authors

Reply:
![image](https://user-images.githubusercontent.com/1797812/90345531-372e5480-dfd6-11ea-821a-4293d9439c75.png)


This actually explains Rules and RuleSets correctly. Maybe we should just rename Rule to Thunk and ThunkSet/ThunkList?